src/printer.js

```diff
@@ -2472,6 +2472,10 @@ function genericPrintNoParens(path, options, print, args) {
       } else if (n.body) {
         parts.push(
           " {",
+          path.call(
+            bodyPath => comments.printDanglingComments(bodyPath, options),
+            "body"
+          ),
           indent(concat([line, group(path.call(print, "body"))])),
           line,
```

Comment:
there are two `line` that are always printed. Probably want to make one only when n.body exists.